src/core/utils.js

```diff
@@ -29,6 +29,18 @@ export const ISODate = new Intl.DateTimeFormat(["en-ca-iso8601"], {
   day: "2-digit",
 });
 
+// We use an "Australian Date" because it omits the ","
+// after the day of the month, which is required by the W3C.
+const dateLang =
+  docLang === "en" || docLang.startsWith("en-") ? "en-AU" : docLang;
+
+export const W3CDate = new Intl.DateTimeFormat(dateLang, {
```

Comment:
As we're calling .format everywhere, maybe we should export a function formatDateW3C that accepts a string|Date instead?

Reply:
Yeah, `.format()` everywhere, but seems like having a wrapper is not necessary. I'd rather just have a single object we work from (checks and conversion are where needed). 

The places where it's doing the wrong thing probably need validateAndRecover()